### ISSUE TITLE
Editing Migration Plan: When importing a CSV that contains VMs already in the plan, they are not selected and have the warning "VM exists in a different migration plan"

### ISSUE THREAD

**Issue by mturley** [ManageIQ/manageiq-v2v#640]
To be addressed after #620 is merged. Here are screenshots from @michaelkro:

<img width="1920" alt="45491970-12216f00-b739-11e8-9a7e-45847aaecfe1" src="https://user-images.githubusercontent.com/811963/45516397-ef16af80-b778-11e8-971a-9843d3a9ecab.png">

<img width="1920" alt="45491470-c7532780-b737-11e8-9468-705d993ebc98" src="https://user-images.githubusercontent.com/811963/45516406-f50c9080-b778-11e8-95be-dae102518347.png">

If the VMs from a CSV are already in the plan, they should be marked as OK to migrate and pre-selected in the VMs step.

**Comment by vconzola** [comment]
Duplicate of #677
